server/src/routes/bitcoin/operational.js

```diff
@@ -23,25 +23,25 @@ router.get('/', (_req, res) => {
  * @swagger
  * /api/v1/bitcoin/operational/storage-constraint:
  *  get:
- *      description: Returns data points for the storage constraint factor in the operational layer for bitcoin (takes a while to test!)
+ *      description: Returns plot points for the storage constraint factor in the operational layer for Bitcoin - TEST EXECUTION MAY BE SLOW
  *      responses:
  *          200:
  *              description: Successful response
  */
 router.get('/storage-constraint', async (_req, res) => {
   const resp = await fetch(
-    'https://charts.bitcoin.com/btc/api/chart/blockchain-size',
+    'https://api.blockchair.com/bitcoin/blocks?a=date,sum(size)',
     { method: 'GET' }
   );
   const respJSON = await resp.json();
+  const respData = respJSON.data;
   const compareInterval = 14; // days
   const data = [{ id: 'Bitcoin', data: [] }];
-  for (let i = compareInterval; i < respJSON.length; i += 1) {
-    const millis = 1000 * respJSON[i][0];
-    const sizeNow = parseInt(respJSON[i][1], 10);
-    const sizeBefore = parseInt(respJSON[i - compareInterval][1], 10);
+  for (let i = compareInterval; i < respData.length; i += 1) {
+    const sizeNow = parseInt(respData[i]['sum(size)'], 10);
+    const sizeBefore = parseInt(respData[i - compareInterval]['sum(size)'], 10);
```

Comment:
```suggestion
    const sizeNow = respData[i]['sum(size)'];
    const sizeBefore = respData[i - compareInterval]['sum(size)'];
```
No need to `parseInt()` as it is already in number form